pkg/awsvpc/examples/out/explain_out/nacl_blocking_all_vpcs_explain_detail.txt

```diff
@@ -4,7 +4,7 @@ Interpreted destination: q2[10.240.32.122]
 ======================================================================
 
 No connections from p2[10.240.2.28] to q2[10.240.32.122];
-connection is blocked by ingress
+	connection is blocked due to ingress
```

Comment:
```suggestion
	connection is blocked at ingress
```
"ingress" is not a mechanism that can block things. It simply means "incoming"

Reply:
done